packages/grafana-ui/src/components/Select/SelectPerf.internal.story.tsx

```diff
@@ -1,13 +1,28 @@
 import { Meta } from '@storybook/react';
+import { fireEvent, getAllByRole } from '@testing-library/dom';
 import { useMemo } from 'react';
 import ReactSelect, { createFilter } from 'react-select';
+import { InteractionTaskArgs, PublicInteractionTask, withPerformance } from 'storybook-addon-performance';
 
 import { SelectableValue } from '@grafana/data';
 
 import { Label } from '../Forms/Label';
 
 import { Select, VirtualizedSelect } from './Select';
 
+const interactionTasks: PublicInteractionTask[] = [
+  {
+    name: 'Search and select an option',
+    description: 'Enter search string and select the first alternative',
+    run: async ({ container }: InteractionTaskArgs): Promise<void> => {
+      const select = getAllByRole(container, 'combobox')[2];
+      select.focus();
+      fireEvent.change(select, { target: { value: '400' } });
```

Comment:
I think we get 566ms because this is a bit simplified. Typing 400 would require three change events IRL.

Reply:
This would work better by using `user-event` https://testing-library.com/docs/user-event/keyboard